rclpy/test/test_node.py

```diff
@@ -838,6 +838,162 @@ def test_node_set_parameters_rejection(self):
         self.assertIsInstance(result[0], SetParametersResult)
         self.assertFalse(result[0].successful)
 
+    def reject_parameter_callback_1(self, parameter_list):
+        rejected_parameters = (param for param in parameter_list if 'reject1' in param.name)
+        return SetParametersResult(successful=(not any(rejected_parameters)))
+
+    def reject_parameter_callback_2(self, parameter_list):
+        rejected_parameters = (param for param in parameter_list if 'reject2' in param.name)
+        return SetParametersResult(successful=(not any(rejected_parameters)))
+
+    def reject_parameter_callback_3(self, parameter_list):
+        rejected_parameters = (param for param in parameter_list if 'reject3' in param.name)
+        return SetParametersResult(successful=(not any(rejected_parameters)))
+
+    def test_node_set_paramters_rejection_list(self):
+        # Declare a new paramters and set a list of callbacks so that it's rejected when set.
+        reject_list_parameter_tuple1 = [
+            (
+                'reject1',
+                True,
+                ParameterDescriptor()
+            ),
```

Comment:
Please keep the code more compact. These file line can easily fit in a single line:

```
('reject1', True, ParameterDescriptor()),
```

Same multiple times below.

Reply:
@suab321321 please address @dirk-thomas's comments here

Reply:
@dirk-thomas okay sir